Fix IndigoDark tooltip

rad-pat opened the pull request:
Remove label appearance which was preventing the tooltip textcolor cascading through.
I can't see that this has upset any other label usage in the Widget Browser

Did look like this:
<img width="96" alt="image" src="https://user-images.githubusercontent.com/45275736/210763480-e30de0cf-fd9f-4846-8c71-2fe3e37adb4d.png">

Now looks like this:
<img width="68" alt="image" src="https://user-images.githubusercontent.com/45275736/210763585-3f064afe-1387-42b7-9bb8-773a0040901a.png">

rad-pat committed:
Remove label appearance which was preventing the tooltip text coloring

oetiker commented:
hmm not sure, but from looking at the patch I would worry that it affects other things too

sknick commented:
> hmm not sure, but from looking at the patch I would worry that it affects other things too. Schouldn't a tooltip in a dark theme be dark anyway ?

That's debatable I guess. I think what rad-pat's commit does is fix another case of a hard-coded color in the Appearance theme, which I had to do several times in making the dark theme.

sknick commented:
Since a version of qx hasn't even been released yet that contains the new theme, there is no chance of regressions for existing qx users, so it probably won't hurt to throw caution to the wind and just merge it. Just my $0.02.

derrell reviewed (approved):
`qx.theme.indigo.Appearance`, which this inherits from does not have a `label` appearance, so this was overwriting the super-superclass `label` in `qx.theme.simple.Appearance` The difference is that when `states.disabled` is falsy, `textColor` is set to `undefined` instead of "text".

The `AppearanceDark` theme, to my knowledge, has never yet seen the light of day in a release, so this can as easily as not simply be called a bug in the original commit of this new appearance.

Good to go. Ship it!

goldim reviewed (changes requested):
I checked out other two dark themes and a tooltip has black/dark background color and white text color there.  My opinion is IndigoDark tooltip also should be like those.

goldim commented:
A theme select button in Widget Browser now has black text color on black background with the change too.

rad-pat committed:
Remove redundant listitem - this was an unchanged copy from simple

rad-pat committed:
Tooltip gray with light text

rad-pat commented:
Updated tooltip to be black with light text. Removed redundant appearance for listitem.
The black text in the selectbox of widget browser will likely need fixing there https://github.com/qooxdoo/qxl.widgetbrowser/blob/a4ad62842f5208b40b2aa3a2283acfda4fbedcf1/source/class/qxl/widgetbrowser/view/Header.js#L68

goldim commented on source/class/qx/theme/indigo/ColorDark.js:
```diff
@@ -85,8 +85,8 @@ qx.Theme.define("qx.theme.indigo.ColorDark", {
     // tooltip
-    "tooltip": "#dddddd",
-    "tooltip-text": "#000000",
+    "tooltip": "#000000",
```
I think less harsh color than a pure black color for background would be nice some kind of gray. Tried #666666 - probably looks better than black.

rad-pat replied:
Updated to gray.
